taxi_service/settings.py

```diff
@@ -112,7 +112,7 @@
 
 LANGUAGE_CODE = "en-us"
 
-TIME_ZONE = "UTC"
+TIME_ZONE = "Europe/Kiev"
```

Comment:
Ensure that the `TIME_ZONE` setting ('Europe/Kiev') is appropriate for your application's requirements. Adjust it if necessary.